app/controllers/admin/registrations_controller.rb

```diff
@@ -12,6 +12,7 @@ def index
 
       @registration_distribution = @conference.registration_distribution
       @affiliation_distribution = @conference.affiliation_distribution
+      @code_of_conduct = !@conference.code_of_conduct.blank?
```

Comment:
use `@conference.code_of_conduct.present?` instead of `!@conference.code_of_conduct.blank?` please :see_no_evil: